UKHomeOffice/sgmr-service
# NMSW 55/56 audit packages

### AC / Description of changes

NMSW-55: Moment
NMSW-56: ansi-regex

Moment is used as a dependency in a cypress package so cannot be removed fully but the version is successfully updated in the package-lock.json (and was already removed from the package.json)

Ansi-regex was also updated when using `npm audit fix`

### To Test

- When on this branch (NMSW-55/56-audit-packages)
- In the terminal run `npm ci`
- Run app locally
- Go to localhost:8080
- The app should load and run correctly and without errors

### Notes

---

**Remember to merge to feature branches not `master`.**
